apps/client/src/pages/ConclusionPage.js

```diff
@@ -99,6 +99,98 @@ const ConclusionPage = ({ topic, checker }) => {
   };
   const currentDateTime = date.toLocaleDateString("nl-NL", options);
 
+  /**
+   * Demo saveCheckerResult function
+   *
+   * This function is called:
+   * - every time the conclusion page is loaded (without user data)
+   * - every time the user wants to save the progress (including user data)
+   * - every time the user wants to save the conclusion (including user data)
+   *
+   * This function is called from another file. This is just a sketch.
+   */
+  const saveCheckerResult = () => {
+
+    const mutationData = {
+
+      environmentUrl: window.location.href,
+      appVersion: process.env.REACT_APP_VERSION,
+      topic: topic,
+
+
+      // How in the world are we going to get STTR version(s)?
+      // In topic.sttrFile we know the JSON file, but that's it
+      // If we really want the STTR version, we can do that 2 ways:
+      // 1) We go into topics.source.json and go get all activity id's and with those go into the xml files
+      // 2) (Re)decide to add them into the [topic].json file
+      activities: [
+        {
+          id: 'monument activity',
+          version: 1.2.4,
```

Comment:
This would give a parse error. Consider making 1.2.4 a string.
Maybe good to double-check if FL is using semantic versioning.

Reply:
Any feedback on "How in the world are we going to get STTR version(s)?"